src/Icon/Icon.tsx

```diff
@@ -21,33 +23,23 @@ export type IconProps = {
   color?: Color
   /** rotation degrees */
   rotate?: number
-  /** margins TOP-RIGHT-BOTTOM-LEFT */
-  mt?: string
-  mr?: string
-  mb?: string
-  ml?: string
```

Comment:
This might be a breaking change

Reply:
yeah I think this will probably break in quite a lot of places